test/lib/util/memcache_test.exs

```diff
@@ -0,0 +1,71 @@
+# Copyright(c) 2015-2018 ACCESS CO., LTD. All rights reserved.
+
+defmodule Antikythera.MemcacheTest do
+  use Croma.TestCase
+  alias AntikytheraCore.ExecutorPool
+  alias AntikytheraCore.ExecutorPool.Setting, as: EPoolSetting
+  alias Antikythera.TermUtil
+
+  @epool_id        {:gear, :testgear}
+  @max_records_num 10
+  @max_key_size    128
+  @max_value_size  65536
```

Comment:
Let's add `def` just to retrieve the constants.
```ex
alias Antikythera.Memcache.{Key, Value}
alias AntikytheraCore.ExecutorPool.MemcacheWriter

@max_records_num MemcacheWriter.max_records()
@max_key_size    Key.max_size()
@max_value_size  Value.max_size()
```

Reply:
fixed in 60b7e30ab96720c852fce613a29e99e0801de475